sdk/eventgrid/Azure.Messaging.EventGrid/README.md

```diff
@@ -323,6 +323,9 @@ You can also easily [enable console logging](https://github.com/Azure/azure-sdk-
 ### Distributed Tracing
 The Event Grid library supports distributing tracing out of the box. In order to adhere to the CloudEvents specification's [guidance](https://github.com/cloudevents/spec/blob/master/extensions/distributed-tracing.md) on distributing tracing, the library will set the `traceparent` and `tracestate` on the `ExtensionAttributes` of a `CloudEvent` when distributed tracing is enabled. To learn more about how to enable distributed tracing in your application, take a look at the Azure SDK [distributed tracing documentation](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/samples/Diagnostics.md#Distributed-tracing).
 
+### Event Grid on Kubernetes 
```

Comment:
I thought the purpose of Arc was to allow a self-hosted service resource on K8s;  wouldn't the client be the same as any other client in a container-hosted application?

Reply:
Yes, the client would be the same - this is just saying that we have tested it and it works.
https://azure.microsoft.com/en-us/updates/public-preview-run-event-grid-on-kubernetes-in-azure-or-anywhere-with-azure-arc/